Prepend PR body defaults to the selected template (if any)

dhleong opened the pull request:
See #1570

Thought I'd go ahead and get the ball rolling with a simple change to address at least part of the issue linked above :)

dhleong committed:
Prepend PR body defaults to the selected template (if any)

See #1570

dhleong committed:
Add a test to verify the behavior of prepending commit body to message

dhleong committed:
"Check" the error value of ioutil.WriteFile

We could, alternatively, use t.Fatal... this matches the MkdirAll
invocation from github_template_test, however...

vilmibm reviewed (changes requested):
ahh, unfortunately I just merged https://github.com/cli/cli/pull/1582 and, after merging `trunk` with this PR, the tests are failing. Sorry about that timing, but once this is reconciled with `trunk` I'm :+1:

dhleong committed:
Update template test to reflect inclusion of "default" text

Also, avoid excessive amounts of newlines separating the two

dhleong commented:
Based against `trunk`, updated the failing test to reflect the inclusion of the "default" body before the template, and also added some quick logic to avoid having more than two newlines separating the template text and the "default" body.

vilmibm reviewed (approved):
thank you!!
